test/common_tests.dart

```diff
@@ -2232,23 +2235,27 @@ void runCommonTests(
             expect(await f.readAsString(), 'Hello world\n');
           });
 
+          // TODO(tvolkert): Fix and re-enable: http://dartbug.com/29554
```

Comment:
Maybe mark this test as skipped instead?

Reply:
We hijack the test() command in this file to enable us to call into this from various file system tests, so skip isn't supported here.